deploy/olm-catalog/eunomia/0.1.0/gitopsconfigs.eunomia.kohls.io.crd.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Same for this file. This file was submitted to repo https://github.com/operator-framework/community-operators. Why do we need this file in our repo?